### PR TITLE
Added documentation for PEM_X509_INFO_read() and PEM_X509_INFO_read_bio()

### PR DESCRIPTION
**Pull request by allanchang96**
Fixes: #18342 

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/PEM_X509_INFO_read_bio_ex.pod [hunk 1]
```diff
@@ -13,19 +13,30 @@ PEM_X509_INFO_read_bio_ex, PEM_X509_INFO_read_ex
...
+ STACK_OF(X509_INFO) *PEM_X509_INFO_read_bio(BIO *bp, STACK_OF(X509_INFO) *sk,
+                                             pem_password_cb *cb, void *u);
 
 =head1 DESCRIPTION
 
 PEM_X509_INFO_read_ex() loads the B<X509_INFO> objects from a file I<fp>.
 
+PEM_X509_INFO_read() is similar to PEM_X509_INFO_read_ex() 
+but uses the default (NULL) library context I<libctx> 
+and property query I<propq>.
```
Comment (paulidale) [inline doc/man3/PEM_X509_INFO_read_bio_ex.pod:32]: and _empty_ property query?

**Review by paulidale — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by paulidale** [comment]
This goes well beyond what is acceptable under a trivial CLA.  Could you please submit a CLA?

**Comment by paulidale** [comment]
There is also some end of line white space, if you could remove them too.

**Comment by t8m** [comment]
And furthermore there are merge commits. Not sure, why we did not notice this before. So please rebase your branch on top of fresh master branch and drop the merge commits.

**Comment by allanchang96** [comment]
Opened up a new PR

https://github.com/openssl/openssl/pull/18623

**Pull request closed without merging**